python/py_helpers.py

```diff
@@ -112,9 +197,28 @@ def has_import(self, import_str):
             import_node.is_equivalent(import_str) for import_node in self.find_imports()
         )
 
+    # find a list of function calls of the 'name' function
+    def find_calls(self, name):
+        call_list = []
+        for node in self._find_all(ast.Expr):
+            if func := getattr(node.tree.value, "func", False):
+                if isinstance(func, ast.Name) and func.id == name:
+                    call_list.append(Node(node.tree.value))
+                elif isinstance(func, ast.Attribute) and func.attr == name:
+                    call_list.append(Node(node.tree.value))
+        return call_list
+
     def has_call(self, call):
         return any(node.is_equivalent(call) for node in self._find_all(ast.Expr))
 
+    def find_call_args(self):
+        if not isinstance(self.tree, ast.Call):
+            return []
+        return [Node(arg) for arg in self.tree.args]
+
+    def has_node(self, node_str):
+        return any(Node(node).is_equivalent(node_str) for node in self.tree.body)
```

Comment:
```suggestion
        if not self._has_body():
            return False
        return any(Node(node).is_equivalent(node_str) for node in self.tree.body)
```
Otherwise this can error if no body.